collector/fixtures/e2e-output.txt

```diff
@@ -800,7 +800,7 @@ node_md_disks{device="md10"} 2
 node_md_disks{device="md11"} 2
 node_md_disks{device="md12"} 2
 node_md_disks{device="md127"} 2
-node_md_disks{device="md219"} 2
```

Comment:
That's not correct either though, right? If I read the fixture correctly, we should export `3` here?